source/PlayerInfo.cpp

```diff
@@ -2112,6 +2112,29 @@ void PlayerInfo::SelectNext()
 
 
 
+void PlayerInfo::DeselectAllSecondaries()
+{
+	selectedWeapons.clear();
+}
+
+
+
+void PlayerInfo::ToggleAnySecondary(const Outfit *outfit)
+{
+	if(!flagship)
+		return;
```

Comment:
Is this to handle being dead?

Reply:
Yes, the method SelectNext() (renamed SelectNextSecondary() in this PR) does something similar (it's used for cycling with the select secondary weapon key) has this check also so it seemed appropriate to also include it here.